src/overview.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: page
+title: Overview
+---
+
+# Overview
+
+Workbox is a collection of libraries and build tools that make it easy to
+store your website's resources locally, on your users' devices. Consider Workbox
+if you want to:
+
+* Make your site work offline.
+* Improve load performance on repeat-visits. You can use Workbox to store
+  and serve common resources locally, rather than from the network.
+
+The only difference between the two options is whether you store all or only
+some resources locally.
+
+## The technology behind Workbox
+
+Modern offline web apps are possible thanks to **service workers**. Service
+workers are background workers, written in JavaScript, that can locally save
+all of the resources needed to run your site offline.
+
+You can think of a service worker as a [proxy server][proxy] that's running in
+the background of your users' devices. Your site makes a request for a resource,
+the service worker intercepts the request, and then decides whether to serve a
+local version of the resource or to fetch a fresh version from the network.
+
+[proxy]: https://en.wikipedia.org/wiki/Proxy_server
+
+The place where Workbox stores resources locally is called the **cache**.
+The act of storing locally is called **caching**. This is because the
+underlying technology that Workbox relies on is called the [Cache API][cache].
+This is completely different than [HTTP caching][http]. The main benefit
```

Comment:
Being able to precache things is definitely a benefit. But as mentioned in the earlier comments, the main benefit from a performance point of view is being able to use a cache-first approach to serving HTML documents, which cuts out the network latency involved in waiting for a server to respond to a navigation request.